### PR TITLE
Add simple edge cloud connect test.

### PR DESCRIPTION
**Pull request by petedyerarm**

### CODE REVIEW

**Comment by petedyerarm** [comment]
synchronize with https://github.com/ARMmbed/mbl-tools/pull/306

File: ci/lava/tests/test-edge-connected.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                for line in log:
+                    if line.find("Endpoint id") != -1:
+                        print(line)
+                        lineFound = True
+
+            if lineFound is False:
+                err = 1
+
+        else:
+            print("Failed to get log file from device")
```
Comment (diego-sueiro) [inline ci/lava/tests/test-edge-connected.py]: In this case, shouldn't `err = 1 `?

Comment (petedyerarm) [reply ci/lava/tests/test-edge-connected.py]: If will already be non-zero to get into this branch, so isn't necessary.
The assert of err == 0 will cause the test to fail.

Comment (diego-sueiro) [reply ci/lava/tests/test-edge-connected.py]: That's true.

**Pull request merged**